pkg/qemu/qemu.go

```diff
@@ -194,10 +208,13 @@ func Cmdline(cfg Config) (string, []string, error) {
 	args = append(args, "-cdrom", filepath.Join(cfg.InstanceDir, filenames.CIDataISO))
 
 	// Network
+	if y.Network.VDE.URL != "" {
+		args = append(args, "-device", fmt.Sprintf("virtio-net-pci,netdev=net0,mac=%s", macAddress(cfg)))
+		args = append(args, "-netdev", fmt.Sprintf("vde,id=net0,sock=%s", y.Network.VDE.URL))
+	}
 	// CIDR is intentionally hardcoded to 192.168.5.0/24, as each of QEMU has its own independent slirp network.
-	// TODO: enable bridge (with sudo?)
 	args = append(args, "-net", "nic,model=virtio")
```

Comment:
This line probably needs netdev=net1

Reply:
Thanks! This fixed the interface ordering issue.